app/src/main/java/ch/epfl/sweng/swenggolf/offer/answer/ListAnswerAdapter.java

```diff
@@ -209,6 +210,10 @@ private Dialog createFavoriteDialog(Context context, final int pos, String title
                 .setMessage(hint)
                 .setPositiveButton(android.R.string.yes, new DialogInterface.OnClickListener() {
                     public void onClick(DialogInterface dialog, int which) {
+                        if (pos != NO_FAVORITE) {
+                            UserStats.updateStat(UserStats.Stats.ANSWERS_ACCEPTED,
```

Comment:
Just by curiosity, does that mean that if an answer is chosen, then the user changes his mind and chooses another one, it is still counted? Maybe that some "malicious" users could take advantage of it to boost their stats...

Reply:
It does indeed.
As discussed at lunch, we will leave it this way as only the user can see his own stats and he can not exploit it himself (as he can not favorite his own answers).
And additionally, the stats are not used for anything meaningful so there is no need to be as rigorous as the reputation points that have a leaderboard.

Reply:
Yes, you convinced me :)